src/main/backend.ts

```diff
@@ -203,23 +204,36 @@ export class MainBackend extends BaseBackend<ElectronOptions> implements CommonB
       forget(this._sendNativeCrashes({}));
     }
 
+    /**
+     * Helper function for sending renderer crashes
+     */
+    const sendRendererCrash = async (contents: Electron.WebContents, details?: Electron.Details) => {
+      try {
+        await this._sendNativeCrashes(this._getRendererElectronContext(contents, details));
+      } catch (e) {
+        console.error(e);
+      }
+
+      addBreadcrumb({
+        category: 'exception',
+        level: Severity.Critical,
+        message: 'Renderer Crashed',
+      });
+    };
+
     // Every time a subprocess or renderer crashes, start sending minidumps
     // right away.
     app.on('web-contents-created', (_, contents) => {
-      contents.on('crashed', async () => {
-        try {
-          await this._sendNativeCrashes(this._getRendererExtra(contents));
-        } catch (e) {
-          console.error(e);
-        }
-
-        addBreadcrumb({
-          category: 'exception',
-          level: Severity.Critical,
-          message: 'Renderer Crashed',
-          timestamp: new Date().getTime() / 1000,
+      // We need to update this once this lands in 9.x
+      if (getElectronVersion().major === 8 && getElectronVersion().minor >= 4) {
```

Comment:
We also need to support `>= v10.x.y`.

If it's safe to assume that the next 9.x release will be 9.2 and will include the backport, we need:

```
getElectronVersion().major === 8 && getElectronVersion().minor >= 4 || 
getElectronVersion().major === 9 && getElectronVersion().minor >= 2 ||
getElectronVersion().major >= 10
```

Reply:
Or you could use `semver` and do:

```typescript
semver.satisfies(process.versions.electron, '>= 10.x || >=9.2 || >= 8.4')
```

Reply:
Better no new deps :|
But will change